site/content/extend/plugins/developer-workflow.md

```diff
@@ -15,7 +15,7 @@ subsection: Plugins
 - `make enable` - Enables the plugin on the Mattermost server
```

Comment:
interesting.  I haven't seen this command.

Reply:
@jfrerich It should be available for all repos sync'd with the starter template. `make reset` is usually more useful though, which is like running `make disable && make enable`